src/api/baseapi.cpp

```diff
@@ -146,6 +146,9 @@ static void ExtractFontName(const char* filename, std::string* fontname) {
  */
 static void addAvailableLanguages(const std::string &datadir,
                                   std::vector<std::string> *langs) {
+  if (!std::filesystem::is_directory(datadir))
```

Comment:
Do we still want this check? My impression is that we agreed to prefer reporting the exception which is thrown without this conditional code.

Reply:
If throwing an exception is the agreed-upon approach, then this check is no longer necessary.